backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5703210_sys_gh16181_add_isTaxIncluded_sap_gljournal_quick_input_layout.sql

```diff
@@ -0,0 +1,7 @@
+-- Add the IsTaxIncluded to the QuickInput
+UPDATE ad_sysconfig
+SET value     = 'PostingSign,GL_Account_ID,Amount,M_SectionCode_ID,IsTaxIncluded?,C_Tax_ID?'
+  , updated   = NOW()
+  , updatedby = 99
+WHERE name = 'SAP_GLJournalLine.quickInput.layout'
```

Comment:
pls use `set_sysconfig_value` db function.
pls check with issue giver/PO if this is just a customer customization or it shall go to metasfresh standard?

Reply:
Confirmed with mark, should go into metasfresh